### PR TITLE
feat(shipyard): do not force containers

### PR DESCRIPTION
**Pull request by jkurz**
* Allow users to create shipments without containers
* immediately direct users to command bridge when their shipment has been created

### CODE REVIEW

File: public/js/src/shipyard/containers.tag [hunk 1]
```diff
@@ -47,43 +58,46 @@
...
-            }
 
-            container.ports.map(function(port) {
-                if (port.external) {
-                    if (!port.healthcheck) {
-                        RiotControl.trigger('flash_message', 'error', port.name + ':External ports must have a Healthcheck.', 30000);
-                        valid = false;
-                    }
-                }
+            if (contianer.ports) {
```
Comment (jtolsma) [inline public/js/src/shipyard/containers.tag]: misspelled container

**Comment by jtolsma** [comment]
What is the use case here?  Could this leave to confusion when trigger does not work?

**Comment by jkurz** [comment]
This is so you don't have to have something in catalog before you create a container. Get's rid of the chicken and egg problem for people. 

Proper messages are displayed to let people know there are no containers on their shipment. Trigger also fails properly as well and that message is shown in the UI.

**Review by wilsonodk — APPROVED** [review]
👍

**Review by jtolsma — APPROVED** [review]

**Pull request merged**